design/src/genre-analysis.md

```diff
@@ -188,6 +194,17 @@ These things make the game loop more pleasant:
   - Prototype and measure designs in a sandbox environment
 - **Production planner**
   - Analyze theoretical performance and ratios of resource pathways
+- **Map**
+  - Summarizes the area visually
+  - Often paired with a small, always-on-screen **minmap**
+  - Augmented with **map markers**, which are player-made indicators of specific locations (ideally text + an icon)
+- **Notes**
```

Comment:
Special case of notes: TODO lists

Can be integrated in the game that items are automatically marked as completed (e.g. when a machine is built) or displaying resource requirements for a built (closely coupled to blueprints)

Reply:
```suggestion
- **Notes**
  - commonly used for TODO lists
```